test/e2e/page-objects/pages/home/activity-list.ts

```diff
@@ -46,7 +46,10 @@ class ActivityListPage {
     tag: 'button',
   };
 
-  private readonly speedupButton = '[data-testid="speedup-button"]';
```

Comment:
ℹ️  an alternative would be to use the other selector `speed-up-button` for clicking the inline button, as we want, but I found that it might be confusing (as it already is, as the 2 buttons have very similar test ids)